presto-main/src/test/java/com/facebook/presto/block/BlockAssertions.java

```diff
@@ -368,6 +371,27 @@ public static Block createArrayBigintBlock(Iterable<? extends Iterable<Long>> va
         return builder.build();
     }
 
+    public static Block createObjectsArrayBlock(Type type, Object... object)
```

Comment:
It looks like `StructuralTestUtil` has equivalent methods already, but is in `presto-tests`. Consider moving it to `presto-main` and using that